src/request/request-tab.js

```diff
@@ -34,7 +34,8 @@ module.exports = {
 // TODO: Need to come up with a better self registration process
 require('./components/request-detail-panel-execution');
 var request = require('./components/request-detail-panel-request');
-require('./components/request-detail-panel-logging');
+var LoggingComponent = require('./components/request-detail-panel-logging').Logging;
+var LoggingViewModel = require('./view-models/LoggingViewModel').LoggingViewModel;
```

Comment:
I wonder if the models should be put in a folder under `components` called `models`. That way it implies that the models are for the component. Plus they don't really have a use outside of the components they apply to.